src/lobsterpy/plotting/__init__.py

```diff
@@ -1234,3 +1235,128 @@ def get_plot(
                 ax.scatter(x, y, s=marker_size, marker=marker_style, label=label)
 
         return plt
+
+
+class BWDFPlotter:
```

Comment:
Could be something for pymatgen in the future once BWDF in Lobster are more established